pkg/flexvolume/flexvolume.go

```diff
@@ -143,127 +144,143 @@ func processOpts(optsStr string) (Options, error) {
 
 // ExecDriver executes the appropriate FlexvolumeDriver command based on
 // recieved call-out.
-func ExecDriver(driver Driver, args []string) {
+func ExecDriver(drivers map[string]Driver, args []string) DriverStatus {
```

Comment:
I suggest we implement the driver selection logic elsewhere and then pass it into `ExecDriver` as before rather than overloading `ExecDriver`.